authnid/make_app.py

```diff
@@ -65,3 +78,18 @@ def _make_crl_validator(app):
 
 def _make_token_manager(app):
     app.token_manager = TokenManager(app.config.get("TOKEN_SECRET"))
+
+
+def _apply_request_hooks(app):
+    @app.before_request
+    def before_request():
+        app.db = connect_db(app.db_uri)
+        cursor = make_cursor(app.db)
+        autocommit = app.config['ENV'] != 'test'
```

Comment:
This is just my inexperience talking, but why don't we want test data to commit? Do we not have any tests that would require data being committed and then checked at another point? I guess I would expect the DB table(s) to be rebuilt for test runs (versus not committing data).